src/Parser/Core/Modules/DamageTaken.js

```diff
@@ -0,0 +1,53 @@
+import Module from 'Parser/Core/Module';
+import { getMagicDescription } from 'common/DamageTypes.js';
+import SPELLS from 'common/SPELLS';
+
+const debug = false;
+
+class DamageValue {
+  Amount = 0;
```

Comment:
In JavaScript it's common practice to always use lower case properties and function names.